### PR TITLE
OSDOCS-6187: Update the OCP 4.14 Welcome page

### PR DESCRIPTION
**Pull request by dfitzmau**
TO DOS:

- Check IBM entries for RT
- Jeana R updates

Was [OSDOCS-6187](https://issues.redhat.com/browse/OSDOCS-6187)
Now [OSDOCS-5986](https://issues.redhat.com/browse/OSDOCS-5986)

Version(s):
4.14 

Link to docs preview:
[Welcome page for 4.14](https://66276--docspreview.netlify.app/openshift-enterprise/latest/welcome/)

QE review:
- [ ] QE has approved this change.

Additional information:
* [GDoc](https://docs.google.com/document/d/181oJDBH5Epzj3teUm39U0Cnl5oKCyRgNpwp0a9QfX9Y/edit#heading=h.3xfnyegx39r4)

### CODE REVIEW

File: welcome/index.adoc [hunk 1]
```diff
@@ -65,43 +69,35 @@ ifdef::openshift-enterprise,openshift-webscale,openshift-origin[]
...
 
-- **xref:../installing/installing_aws/installing-aws-account.adoc#installing-aws-account[Install a cluster on AWS]**: You have many installation options when you deploy a cluster on Amazon Web Services (AWS). You can deploy clusters with xref:../installing/installing_aws/installing-aws-default.adoc#installing-aws-default[default settings] or xref:../installing/installing_aws/installing-aws-customizations.adoc#installing-aws-customizations[custom AWS settings].
-You can also deploy a cluster on AWS infrastructure that you provisioned yourself. You can modify the provided xref:../installing/installing_aws/installing-aws-user-infra.adoc#installing-aws-user-infra[AWS CloudFormation templates] to meet your needs.
+- **xref:../installing/installing_aws/preparing-to-install-on-aws.adoc#preparing-to-install-on-aws[Install a cluster on AWS]**: On AWS, you can install {product-title} on either installer-provisioned infrastructure or user-provisioned infrastructure.
```
Comment (tmalove) [inline welcome/index.adoc:76]:
```suggestion
- **xref:../installing/installing_aws/preparing-to-install-on-aws.adoc#preparing-to-install-on-aws[Install a cluster on AWS]**: On AWS, you can install {product-title} on installer-provisioned infrastructure or user-provisioned infrastructure.
```

**Review by tmalove — APPROVED** [review]

File: welcome/index.adoc [hunk 2]
```diff
@@ -234,12 +232,12 @@ endif::[]
...
 of the {product-title} {product-version} control plane. See how {product-title} control plane and worker nodes are managed and updated through the xref:../machine_management/creating_machinesets/creating-machineset-aws.adoc#machine-api-overview_creating-machineset-aws[Machine API] and xref:../architecture/control-plane.adoc#operators-overview_control-plane[Operators].
 
-- **Enable cluster capabilities that were disabled prior to installation** Cluster administrators can enable cluster capabilities that were disabled prior to installation. For more information, see xref:../post_installation_configuration/enabling-cluster-capabilities.adoc#enabling-cluster-capabilities[Enabling cluster capabilities].
+- **xref:../post_installation_configuration/enabling-cluster-capabilities.adoc#enabling-cluster-capabilities[Enable cluster capabilities]**: As a cluster administrators, you can can enable cluster capabilities that were disabled prior to installation. For more information, see xref:../post_installation_configuration/enabling-cluster-capabilities.adoc#enabling-cluster-capabilities[Enabling cluster capabilities].
```
[the remark this answers is not in the document]
Comment (dfitzmau) [reply welcome/index.adoc:240]: Good catch. No time for the can-can!

File: welcome/index.adoc [hunk 3]
```diff
@@ -65,43 +69,35 @@ ifdef::openshift-enterprise,openshift-webscale,openshift-origin[]
...
 
-- **xref:../installing/installing_azure/installing-azure-account.adoc#installing-azure-account[Install a cluster on Azure]**: You can deploy clusters with xref:../installing/installing_azure/installing-azure-default.adoc#installing-azure-default[default settings], xref:../installing/installing_azure/installing-azure-customizations.adoc#installing-azure-customizations[custom Azure settings], or xref:../installing/installing_azure/installing-azure-network-customizations.adoc#installing-azure-network-customizations[custom networking settings] in Microsoft Azure. You can also provision {product-title} into an xref:../installing/installing_azure/installing-azure-vnet.adoc#installing-azure-vnet[Azure Virtual Network] or use xref:../installing/installing_azure/installing-azure-user-infra.adoc#installing-azure-user-infra[Azure Resource Manager Templates] to provision your own infrastructure.
+- **xref:../installing/installing_aws/installing_azure/preparing-to-install-on-azure#preparing-to-install-on-azure[Install a cluster on Azure]**: On Microsoft Azure, you can install {product-title} on installer-provisioned infrastructure or user-provisioned infrastructure.
```
Comment (jeana-redhat) [inline welcome/index.adoc:78]:
```suggestion
- **xref:../installing/installing_azure/preparing-to-install-on-azure.adoc#preparing-to-install-on-azure[Install a cluster on Azure]**: On Microsoft Azure, you can install {product-title} on installer-provisioned infrastructure or user-provisioned infrastructure.
```

File: welcome/index.adoc [hunk 4]
```diff
@@ -65,43 +69,35 @@ ifdef::openshift-enterprise,openshift-webscale,openshift-origin[]
...
 
-- **xref:../installing/installing_azure_stack_hub/preparing-to-install-on-azure-stack-hub.adoc#preparing-to-install-on-azure-stack-hub[Install a cluster on Azure Stack Hub]**: You can install {product-title} on Azure Stack Hub on installer-provisioned infrastructure.
+- **xref:../installing/installing_azure_stack_hub/preparing-to-install-on-azure-stack-hub.adoc#preparing-to-install-on-azure-stack-hub[Install a cluster on Azure Stack Hub]**: On Microsoft Azure Stack, you can install {product-title} on installer-provisioned infrastructure or user-provisioned infrastructure.
```
Comment (jeana-redhat) [inline welcome/index.adoc:80]:
```suggestion
- **xref:../installing/installing_azure_stack_hub/preparing-to-install-on-azure-stack-hub.adoc#preparing-to-install-on-azure-stack-hub[Install a cluster on Azure Stack Hub]**: On Microsoft Azure Stack Hub, you can install {product-title} on installer-provisioned infrastructure or user-provisioned infrastructure.
```

File: welcome/index.adoc [hunk 5]
```diff
@@ -65,43 +69,35 @@ ifdef::openshift-enterprise,openshift-webscale,openshift-origin[]
...
 
-- **xref:../installing/installing_gcp/installing-gcp-account.adoc#installing-gcp-account[Install a cluster on GCP]**: You can deploy clusters with xref:../installing/installing_gcp/installing-gcp-default.adoc#installing-gcp-default[default settings] or xref:../installing/installing_gcp/installing-gcp-customizations.adoc#installing-gcp-customizations[custom GCP settings] on Google Cloud Platform (GCP). You can also perform a GCP installation where you provision your own infrastructure.
+- **xref:../installing/installing_azure_stack_hub//installing_gcp/preparing-to-install-on-gcp.adoc#preparing-to-install-on-gcp[Install a cluster on GCP]**: On Google Cloud Platform (GCP) you can install {product-title} on installer-provisioned infrastructure or user-provisioned infrastructure.
```
Comment (jeana-redhat) [inline welcome/index.adoc:82]:
```suggestion
- **xref:../openshift-enterprise/latest/installing/installing_gcp/preparing-to-install-on-gcp.adoc#preparing-to-install-on-gcp[Install a cluster on GCP]**: On Google Cloud Platform (GCP) you can install {product-title} on installer-provisioned infrastructure or user-provisioned infrastructure.
```

File: welcome/index.adoc [hunk 6]
```diff
@@ -141,46 +139,46 @@ endif::openshift-origin[]
 xref:../installing/installing_aws/installing-aws-vpc.adoc#installing-aws-vpc[AWS] or
 xref:../installing/installing_gcp/installing-gcp-vpc.adoc#installing-gcp-vpc[GCP] or an existing
 xref:../installing/installing_azure/installing-azure-vnet.adoc#installing-azure-vnet[VNet]
-on Azure, you can install a cluster. You can install a cluster in xref:../installing/installing_gcp/installing-gcp-shared-vpc.adoc#installation-gcp-shared-vpc-prerequisites_installing-gcp-shared-vpc[Installing a cluster on GCP into a shared VPC]
+on Microsoft Azure, you can install a cluster. Also consider in xref:../installing/installing_gcp/installing-gcp-shared-vpc.adoc#installation-gcp-shared-vpc-prerequisites_installing-gcp-shared-vpc[Installing a cluster on GCP into a shared VPC]
```
Comment (jeana-redhat) [inline welcome/index.adoc:142]:
```suggestion
on Microsoft Azure, you can install a cluster. Also consider xref:../installing/installing_gcp/installing-gcp-shared-vpc.adoc#installation-gcp-shared-vpc-prerequisites_installing-gcp-shared-vpc[Installing a cluster on GCP into a shared VPC]
```

File: welcome/index.adoc [hunk 7]
```diff
@@ -141,46 +139,46 @@ endif::openshift-origin[]
...
 
 - **xref:../architecture/understanding-development.adoc#understanding-development[Understand {product-title} development]**: Learn the different types of containerized applications, from simple containers to advanced Kubernetes deployments and Operators.
 
 - **xref:../applications/projects/working-with-projects.adoc#working-with-projects[Work with projects]**: Create projects from the {product-title} web console or OpenShift CLI (`oc`) to organize and share the software you develop.
 
-- **xref:../applications/creating_applications/odc-creating-applications-using-developer-perspective.adoc#odc-creating-applications-using-developer-perspective[Work with applications]**.
+- **xref:../applications/creating_applications/odc-creating-applications-using-developer-perspective.adoc#odc-creating-applications-using-developer-perspective[Creating applications using the *Developer perspective]**: Use the *Developer* perspective in the {product-title} web console to easily create and deploy applications.
```
Comment (jeana-redhat) [inline welcome/index.adoc:169]:
```suggestion
- **xref:../applications/creating_applications/odc-creating-applications-using-developer-perspective.adoc#odc-creating-applications-using-developer-perspective[Creating applications using the Developer perspective]**: Use the *Developer* perspective in the {product-title} web console to easily create and deploy applications.
```

File: welcome/index.adoc [hunk 8]
```diff
@@ -141,46 +139,46 @@ endif::openshift-origin[]
...
 
-- **Use the xref:../web_console/web-console-overview.adoc#about-developer-perspective_web-console-overview[*Developer perspective*]** in the {product-title} web console to
+- **Use the xref:../web_console/web-console-overview.adoc#about-developer-perspective_web-console-overview[*Developer* perspective]** in the {product-title} web console to
 xref:../applications/creating_applications/odc-creating-applications-using-developer-perspective.adoc#odc-creating-applications-using-developer-perspective[create and deploy applications].
 
 - **Use the xref:../applications/odc-viewing-application-composition-using-topology-view.adoc#odc-viewing-application-composition-using-topology-view[*Topology*] view**
 to see your applications, monitor status, connect and group components, and modify your code base.
```
Comment (jeana-redhat) [inline welcome/index.adoc:177]: These seem to be redundant with the two right above

File: welcome/index.adoc [hunk 9]
```diff
@@ -234,12 +232,12 @@ endif::[]
...
 of the {product-title} {product-version} control plane. See how {product-title} control plane and worker nodes are managed and updated through the xref:../machine_management/creating_machinesets/creating-machineset-aws.adoc#machine-api-overview_creating-machineset-aws[Machine API] and xref:../architecture/control-plane.adoc#operators-overview_control-plane[Operators].
 
-- **Enable cluster capabilities that were disabled prior to installation** Cluster administrators can enable cluster capabilities that were disabled prior to installation. For more information, see xref:../post_installation_configuration/enabling-cluster-capabilities.adoc#enabling-cluster-capabilities[Enabling cluster capabilities].
+- **xref:../post_installation_configuration/enabling-cluster-capabilities.adoc#enabling-cluster-capabilities[Enable cluster capabilities]**: As a cluster administrators, you can enable cluster capabilities that were disabled prior to installation. For more information, see xref:../post_installation_configuration/enabling-cluster-capabilities.adoc#enabling-cluster-capabilities[Enabling cluster capabilities].
```
Comment (jeana-redhat) [inline welcome/index.adoc:240]:
```suggestion
- **xref:../post_installation_configuration/enabling-cluster-capabilities.adoc#enabling-cluster-capabilities[Enable cluster capabilities]**: As a cluster administrator, you can enable cluster capabilities that were disabled prior to installation. For more information, see xref:../post_installation_configuration/enabling-cluster-capabilities.adoc#enabling-cluster-capabilities[Enabling cluster capabilities].
```

Comment (jeana-redhat) [inline welcome/index.adoc:240]: Same link twice, with almost identical text. Probably just need one.
```suggestion
- **xref:../post_installation_configuration/enabling-cluster-capabilities.adoc#enabling-cluster-capabilities[Enable cluster capabilities]**: As a cluster administrators, you can enable cluster capabilities that were disabled prior to installation.
```

**Review by jeana-redhat — COMMENTED** [review]
Found a couple broken xrefs and typos, overall looking great!

/remove-label merge-review-in-progress
/remove-label merge-review-needed

File: welcome/index.adoc [hunk 10]
```diff
@@ -65,43 +69,35 @@ ifdef::openshift-enterprise,openshift-webscale,openshift-origin[]
...
+- **xref:../installing/installing_azure_stack_hub/preparing-to-install-on-azure-stack-hub.adoc#preparing-to-install-on-azure-stack-hub[Install a cluster on Azure Stack Hub]**: On Microsoft Azure Stack Hub, you can install {product-title} on installer-provisioned infrastructure or user-provisioned infrastructure.
 
-- **xref:../installing/installing_gcp/installing-gcp-account.adoc#installing-gcp-account[Install a cluster on GCP]**: You can deploy clusters with xref:../installing/installing_gcp/installing-gcp-default.adoc#installing-gcp-default[default settings] or xref:../installing/installing_gcp/installing-gcp-customizations.adoc#installing-gcp-customizations[custom GCP settings] on Google Cloud Platform (GCP). You can also perform a GCP installation where you provision your own infrastructure.
+- **xref:../installing/installing_gcp/installing-gcp-account.adoc#installing-gcp-account[Install a cluster on GCP]**: On Google Cloud Platform (GCP) you can install {product-title} on installer-provisioned infrastructure or user-provisioned infrastructure.
```
Comment (jeana-redhat) [inline welcome/index.adoc:82]: The other links go to the "Preparing to install" page, this one goes to "Configuring a GCP project". Think you want:
```suggestion
- **xref:../installing/installing_gcp/preparing-to-install-on-gcp.adoc#preparing-to-install-on-gcp[Install a cluster on GCP]**: On Google Cloud Platform (GCP) you can install {product-title} on installer-provisioned infrastructure or user-provisioned infrastructure.
```

File: welcome/index.adoc [hunk 11]
```diff
@@ -141,46 +139,40 @@ endif::openshift-origin[]
...
+on Microsoft Azure, you can install a cluster. Also consider xref:../installing/installing_gcp/installing-gcp-shared-vpc.adoc#installation-gcp-shared-vpc-prerequisites_installing-gcp-shared-vpc[Installing a cluster on GCP into a shared VPC]
 
 - **Install a private cluster**: If your cluster does not require external
 internet access, you can install a private cluster on
 xref:../installing/installing_aws/installing-aws-private.adoc#installing-aws-private[AWS],
 xref:../installing/installing_azure/installing-azure-private.adoc#installing-aws-private[Azure],
 xref:../installing/installing_gcp/installing-gcp-private.adoc#installing-gcp-private[GCP], or
-xref:../installing/installing_ibm_cloud_public/preparing-to-install-on-ibm-cloud.adoc#preparing-to-install-on-ibm-cloud[IBM Cloud VPC]. Internet access is still required to access the cloud APIs and installation media.
+xref:../installing/installing_ibm_cloud_public/preparing-to-install-on-ibm-cloud.adoc#preparing-to-install-on-ibm-cloud[{ibmcloudVPCRegProductName} Cloud VPC]. Internet access is still required to access the cloud APIs and installation media.
```
Comment (jeana-redhat) [inline welcome/index.adoc:149]:
```suggestion
xref:../installing/installing_ibm_cloud_public/preparing-to-install-on-ibm-cloud.adoc#preparing-to-install-on-ibm-cloud[{ibmcloudVPCRegProductName}]. Internet access is still required to access the cloud APIs and installation media.
```

**Review by jeana-redhat — COMMENTED** [review]
Ran through the links pre-merge, one seems to have a different pattern and noticed one issue with an attribute.

**Pull request merged**